homeassistant/components/zha/core/cluster_handlers/homeautomation.py

```diff
@@ -65,29 +65,56 @@ class MeasurementType(enum.IntFlag):
         POWER_QUALITY_MEASUREMENT = 256
 
     REPORT_CONFIG = (
-        AttrReportConfig(attr="active_power", config=REPORT_CONFIG_OP),
-        AttrReportConfig(attr="active_power_max", config=REPORT_CONFIG_DEFAULT),
-        AttrReportConfig(attr="apparent_power", config=REPORT_CONFIG_OP),
-        AttrReportConfig(attr="rms_current", config=REPORT_CONFIG_OP),
-        AttrReportConfig(attr="rms_current_max", config=REPORT_CONFIG_DEFAULT),
-        AttrReportConfig(attr="rms_voltage", config=REPORT_CONFIG_OP),
-        AttrReportConfig(attr="rms_voltage_max", config=REPORT_CONFIG_DEFAULT),
-        AttrReportConfig(attr="ac_frequency", config=REPORT_CONFIG_OP),
-        AttrReportConfig(attr="ac_frequency_max", config=REPORT_CONFIG_DEFAULT),
+        AttrReportConfig(
+            attr=homeautomation.ElectricalMeasurement.AttributeDefs.active_power.name,
```

Comment:
I think we should import the ElectricalMeasurement object directly at the top to reduce length.

Reply:
Applies to all clusters i think (with no clashing names)

Reply:
I went back and forth on this. There are a lot of overlapping names and then I went with consistency instead of shortening them. Open to changing it though.

Reply:
what about something like this (but only for handlers with actual implementations) 
![image](https://github.com/home-assistant/core/assets/1335687/1f492d59-09f8-4129-b282-f61e58457bf1)

Reply:
Most cases in a single .py don't seem to overlap much. I think just handle the case when they overlap rather than rename all.

Reply:
The names of the current cluster handlers are a bit inconsistent:
- `ColorClusterHandler`, `IASZoneClusterHandler` `ThermostatClusterHandler`, ... vs
- `Metering`, `RelativeHumidity`, ...

The zigpy clusters classes are all just named after the ZCL cluster, e.g. `Metering`. When that naming scheme is also used for the cluster handler classes, it obviously conflicts a bit.

So, I do wonder if we should think about renaming the conflicting cluster handlers to have them differ from the zigpy cluster names. I think this would allow importing the zigpy cluster classes with no overlaps.

Personally, I also prefer having a "`ClusterHandler` suffix" (compared to using just the cluster name).
(`Handler` might be enough of a suffix though?)

On the other side, being able to shorten `SmartEnergy.AttributeDefs.attribute.id` to something like `SEAttrs.attribute.id` might still be nice?

Reply:
I agree on adding the ClusterHandler suffix on all of the cluster handlers but that can be done in a separate PR

Reply:
I think `general.OnOff` and `general.LevelControl` could be shortened without any conflicts too, and I guess they have a small implementation each.

But I'm fine with this either way.